Sources/UIPilot/UIPilot.swift

```diff
@@ -36,7 +36,7 @@ public class UIPilot<T: Equatable>: ObservableObject {
         }
     }
     
-    public func popTo(_ route: T, inclusive: Bool = false) {
+    public func popTo(_ route: T, inclusive: Bool = false, showIntermediateTransitions: Bool = false) {
         logger.log("UIPilot: Popping route \(route).")
```

Comment:
Another question: the way I've implemented this makes it the default behavior for all clients of the library. And changing that behavior requires passing `showIntermediateTransitions: true`. Is that ok? If not, I could make the default `true`. The downside for me would be the need to pass that arg with every call to `popTo()`.

As an alternative, UIPilot.init could accept a default parameter for this.

Reply:
Thanks for the change Jack, it looks much cleaner now. About flag, I don't think anyone would be interested to show intermediate transitions. If they want, they can always do it themselves with a for loop. Maybe we can make "showIntermediateTransitions: false" the default behavior and can get rid of the flag, what are your thoughts?

On the side note, I'm going to add an API to expose current path of the UIPilot. That will give caller much more control.

Reply:
I agree that the desire to view the intermediate transitions would be unusual. So, I updated the change to remove the flag and make that behavior the default. Thanks!